### DISCUSSION TITLE
Customization of template

### DISCUSSION THREAD

**Discussion by appelgriebsch** [LazyVim/LazyVim#20] [Q&A] [answered]
At the moment I have a fork of this repo to customize some parts of it (e.g. by rearranging key maps or replacing some plug-ins/ settings with my preferred ones)…

keeping my fork updated with changes made in this repo will likely get more complex over time. I noticed in a comment from @folke that there is a way to build an own LazyVim template with the option to **inherit**  from this one and then do your customization (https://github.com/LazyVim/LazyVim/issues/7#issuecomment-1370198940), but I’m still not sure I got all the requirements for it…

If someone can elaborate the concept a bit more in detail or link to some documentation for it, would be very helpful.

Many Thanks @folke for the work on this.

**Comment by folke** [comment, marked as answer]
Still working on all the docs, but you can find the actual template here: https://github.com/LazyVim/starter

That way you no longer need to fork it.

My own config is based on that template as well.

**Comment by folke** [comment]
Since this is all still a work in progress, would be great to get any feedback on using LazyVim with that starter template!

**Reply by appelgriebsch** [reply]
Will give it a try over the weekend…

**Reply by appelgriebsch** [reply]
hmmm when using the starter template in a fresh repo (https://github.com/appelgriebsch/Nv) I receive the following error (Neovim 0.8.1):

```
Error detected while processing /home/.../.config/nvim/init.lua:
No specs found for module lazyvim.plugins
```

does it require a newer Neovim version?

**Reply by appelgriebsch** [reply]
btw: I checked the content of $HOME/.local/share/nvim/lazy/LazyVim and it looks pretty good

**Reply by folke** [reply]
You're probably using an older lazy.nvim version. Try deleting that

**Reply by appelgriebsch** [reply]
ah yes, its not sufficient to remove the one in $HOME/.local/share/nvim/lazy/, I also had to delete the one in $HOME/.cache/nvim/lazy/....

its working now. Thanks for the hint!

**Reply by jrnxf** [reply]
I've really enjoyed using the starter! My only gripe is it can be annoying to have to constantly come back to this repo to check what plugins exist and what their default option are so that I can change them if I need. Not sure if you have any ideas there!

**Reply by folke** [reply]
@jrnxf all plugins are documented on the website https://lazyvim.github.io/configuration/plugins

You can use the search top-left to find the plugin and then you'll see the default options and the full scpe.
